### PR TITLE
[13.0][FIX] mail_tracking: Store cc

### PR DESCRIPTION
**Pull request by Tardo**
Fixes an error when saving "mail_cc" and "mail_to" values in the message.

cc @tecnativa TT28517

### CODE REVIEW

File: mail_tracking/models/mail_thread.py [hunk 1]
```diff
@@ -29,17 +29,24 @@ def _get_failed_message_domain(self):
...
         Because Odoo implementation avoid store 'from, to, cc' recipients we
         ensure that this information its written into the mail.message record.
         """
-        kwargs.update(
-            {"email_cc": kwargs.get("cc", False), "email_to": kwargs.get("to", False)}
+        thread_id = super()._message_route_process(message, message_dict, routes)
+        message = self.env["mail.message"].search(
+            [("message_id", "=", message_dict.get("message_id"))]
+        )
+        message.write(
```
Comment (pedrobaeza) [inline mail_tracking/models/mail_thread.py]: Can we avoid this double writing? This one is critical due to the high volume of the messages, so we shouldn't write several times and do it at message creation.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Check also Travis

**Comment by Tardo** [comment]
Thanks! changes done :)

**Review by pedrobaeza — APPROVED** [review]

**Review by yajo — APPROVED** [review]

**Pull request merged**